test/modules/linear_boltzmann_solvers/dsa/acceleration_diffusion_cfem.cc

```diff
@@ -153,8 +153,9 @@ acceleration_Diffusion_CFEM(const InputParameters&)
   // Make solver
   lbs::DiffusionPWLCSolver solver(
     "SimTest92b_DSA_PWLC", sdm, OneDofPerNode, bcs, matid_2_xs_map, unit_cell_matrices, true);
-  solver.options.ref_solution_lua_function = "MMS_phi";
-  solver.options.source_lua_function = "MMS_q";
+  // TODO: For this to work, add MMS support into `lbs/acceleration/DiffusionSolver`
+  // solver.options.ref_solution_lua_function = "MMS_phi";
+  // solver.options.source_lua_function = "MMS_q";
```

Comment:
This was set, but had no effect, since there is no code in `DiffusionPWLCSolver` to use the MMS-related functions.

Since this was in the code, I assumed this was a wanted capability, therefore I added the `TODO` comment. This can be removed, if needed.

Reply:
leave it. We need to add more MMS. I'll create an issue.